app/logic/Calendar/Invitation.ts

```diff
@@ -23,12 +27,12 @@ export enum ResponseType {
   NoResponseReceived = 5,
```

Comment:
```suggestion
  /** The participant has not yet answered.
   For incoming invitations only. */
  NoResponseReceived = 5,
```

Reply:
I'd have to double-check to see whether this status can appear directly on events. It appears on a participant of a meeting you organise to show that you have not yet seen a response from that participant.

It's also used for your status as an attendee on incoming meeting requests, since at this point you haven't responded yet.